content/docs/learning-guide/tutorials/assets/actor-assets.md

```diff
@@ -0,0 +1,92 @@
+
+{{< note >}}
+Understanding the [best practices](/docs/user-guide/assets/scene-settings/source-asset-best-practices#actors) for creating actor source assets can mitigate issues you might encounter when processing actors for O3DE. For technical details about the data supported by actors, refer to the [supported 3D scene data](/docs/user-guide/assets/scene-settings/scene-format-support#supported-3d-scene-data) table.
+{{< /note >}}
+
+| O3DE Experience | Time to Complete | Feature Focus | Last Updated |
+| - | - | - | - |
+| Beginner | 10 Minutes | Customized processing of actor assets from `.fbx` files with Scene Settings. | January 4, 2023 |
+
+## Preparation
+
+Place your actor `.fbx` source asset in a [scan directory](/docs/user-guide/assets/pipeline/scan-directories) for your project, such as the `Assets` subdirectory, so it can be detected by [**Asset Processor**](/docs/user-guide/assets/asset-processor).
+
+## Processing an actor asset
+
+When you placed your actor `.fbx` source asset in your project's `Assets` directory, Asset Processor detected the asset, examined its contents, and processed it with a default set of rules. These default rules might be sufficient for simple actor assets, however, actors often require additional processing for custom normals and tangents, cloth meshes, root motion extraction, or coordinate space changes. To customize the rules that process the actor asset, do the following:
+
+1. In **O3DE Editor**, locate your asset in **Asset Browser**. You can type the name of the asset into the search field at the top of Asset Browser to filter the list and locate your actor asset.
+
+    ![ Search for a specific mesh asset in Asset Browser. ](/images/learning-guide/tutorials/assets/actor-search-asset-browser.png)
+
+    If your asset has already been processed, you might see a preview image of the asset, and a list of product assets below the `.fbx` source asset.
+
+1. **Double-click** the `.fbx` source asset and choose **Edit Settings...** from the context menu to open Scene Settings.
+
+    ![ Open Scene Settings from Asset Browser. ](/images/learning-guide/tutorials/assets/actor-edit-settings.png)
+
+1. Select the [**Meshes**](/docs/user-guide/assets/scene-settings/meshes-tab) tab.
+
+    ![ Scene Settings meshes tab. ](/images/learning-guide/tutorials/assets/actor-mesh-scene-settings.png)
+
+    In the preceding image, there is one **Mesh group** in the **Meshes** tab. A mesh group produces a runtime optimized mesh asset. By default, all the meshes in the source asset are selected for the mesh group and are processed as a single mesh.
+
+    Additional mesh groups can be created by choosing **Add another mesh**. However, an actor entity can only have one mesh component. All of the actor's meshes - which is to say, all the meshes that are bound to the actor's skeleton - must be included in a single mesh group. The actor in this example has one mesh, though it's common for actors to have many unique meshes.
```

Comment:
We should probably just disable the ability to add a second mesh when actor tab is present.